repository-data/webfiles/src/main/resources/site/freemarker/common/search-results-layout.ftl

```diff
@@ -15,9 +14,8 @@
                     </div>
                 </div>
             </div>
-
             <div class="column column--two-thirds page-block page-block--main">
-                <@searchTabsComponent contentNames=hstResponseChildContentNames></@searchTabsComponent>
+                <@hst.include ref="tabs" />
```

Comment:
Is there a reason for still including the tabs?

Reply:
I will remove this.